0010-endpoint-metadata-extension/README.md

```diff
@@ -292,10 +192,6 @@ The request payload MUST be a UTF-8 encoded JSON object with the JSON Schema def
   "description": "10/EPMDX get metadata request",
   "type": "object",
   "properties": {
-    "id": {
-      "type": "integer",
-      "description": "The message identifier used to match server response to the request"
-    },
     "keys": {
```

Comment:
I thought about that but decided to leave `keys` field as is. That will make it easier to extend the call with other filters if we decide to.